pages/national_resources/si_resources.md

```diff
@@ -3,8 +3,10 @@
 title: Slovenia
 search_exclude: true
 country_code: SI
-contributors: [<!---REPLACE THIS with comma separated list of contributors--->]
-coordinators: [<!---REPLACE THIS with the name of data management coordinators of your ELIXIR node--->]
+page_id:  si_resources
+description: This page provides an overview of resources maintained by ELIXIR Slovenia for the life science research community in the country.
+contributors: [Brane Leskošek, Marko Vidak, Nadja Žlender]
+coordinators: [Brane Leskošek, Marko Vidak, Nadja Žlender]
```

Comment:
I am commenting on the wrong line, but if a few lines down you don't want to specify any specific training from Slovenia, you can delete that specific block.